site/content/docs/5.3/migration.md

```diff
@@ -74,11 +75,17 @@ Learn more by reading the new [color modes documentation]({{< docsref "/customiz
   - The examples in our docs are now explicitly initialized and mostly don't use anymore `data-bs-ride`. Depending on how carousels are used in your project, it might need some updates.
   - <span class="badge bg-danger">Breaking</span> The pause/play button is now included within the carousel for a better rendering but still respecting accessibility guidelines; it is now placed on the left-hand side of the indicators. The HTML markup has changed, please use this new version on your websites.
 
+- **Close button**
+  - A tooltip was added on close button in code examples. Please reflect this modification in your websites.
```

Comment:
Might want to use a "New" label here and maybe provide a few `data-bs` to add to make it work. And say that everywhere users include a close button, this tooltip must be present.